visual-inspection/inspection-frontend/src/components/ExplainInspection.vue

```diff
@@ -1,16 +1,38 @@
 <template>
-  <div>
-    <div class="explanation-request">
-      <button class="xd-button xd-primary"
-              v-show="!waitingForExplanation"
-              v-bind:disabled="!predictionReady"
-              v-on:click="buttonClicked">
-        Woran erkennst du das?
-      </button>
-      <MultiBounce v-if="waitingForExplanation"
-                   v-bind:numberOfDots="3"/>
+    <div>
+        <br />
```

Comment:
```suggestion
```
There is (almost) never a need to use the `br` tag, especially not outside of text paragraphs.